webauthn/createcredential-cross-origin-iframe.https.sub.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WebAuthn credential.create() in a cross-origin iframe tests</title>
+<meta name="timeout" content="long">
+<link rel="help" href="https://w3c.github.io/webauthn/#publickey-credentials-create-feature">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src=helpers.js></script>
+<body></body>
+<script>
+standardSetup(function() {
+    "use strict";
+
+    const targetOrigin = "https://{{hosts[alt][www]}}:{{ports[https][0]}}";
+
+    // Returns a |Promise| that gets resolved with |event.data| when |window|
+    // receives a "message" event whose |event.data.type| matches the string
+    // |message_data_type|.
+    function getMessageData(message_data_type) {
+        return new Promise(resolve => {
+            function waitAndRemove(e) {
+                if (!e.data || e.data.type != message_data_type)
+                    return;
+                window.removeEventListener("message", waitAndRemove);
+                resolve(e.data);
+            }
+            window.addEventListener("message", waitAndRemove);
+        });
+    }
+
+    function createIframe(test) {
+        const iframeElement = document.createElement("iframe");
+        document.body.appendChild(iframeElement);
+        test.add_cleanup(() => {
+            iframeElement.remove();
+        });
+        return iframeElement;
```

Comment:
Consider making `createIframe` take `src` and `allow`, and return a promise that resolves to the `iframe` once `subframe-loaded` is received. That way you don't have add code to wait for the iframe to load on every test.

Reply:
Done, thanks!